.mergify.yml

```diff
@@ -4,6 +4,13 @@ queue_rules:
     merge_conditions:
       # Conditions to get out of the queue (= merged)
       - or:
+          - and:
+              - label=ci/skip/e2e
+              - "status-success=codespell"
+              - "status-success=go-test"
+              - "status-success=golangci-lint"
+              - "status-success=mod-check"
+              - "status-success=uncommitted-code-check"
```

Comment:
`multi-arch-build` and `lint-extras` are missing from the list, those should be included.